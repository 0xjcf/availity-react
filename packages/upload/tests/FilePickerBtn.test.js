import React from 'react';
import renderer from 'react-test-renderer';
import { FilePickerBtn } from '..';

describe('Upload', () => {
  test('should render', () => {
    const component = renderer.create(<FilePickerBtn onChange={() => {}} />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
