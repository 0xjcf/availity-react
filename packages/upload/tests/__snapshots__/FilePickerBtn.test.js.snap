// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload should render 1`] = `
Array [
  <div
    className="d-none"
  >
    <input
      className="form-control-file"
      id="filePicker-1"
      onChange={[Function]}
      type="file"
      value=""
    />
  </div>,
  <button
    aria-label={null}
    className="btn btn-primary"
    onClick={[Function]}
    type="button"
  >
    Select File
  </button>,
]
`;
